doc/conf.py

```diff
@@ -273,6 +273,37 @@ def update_versions(app, docname, source):
         "{{VIZZU_VERSION}}" : VIZZU_VERSION,
     }
 
+def update_versions(app, docname, source):
```

Comment:
Seems to duplicate the code from just above.

Reply:
Fixed the duplication @philippjfr .